src/transformers/models/speecht5/modeling_speecht5.py

```diff
@@ -2541,6 +2541,15 @@ def _generate_speech(
     vocoder: Optional[nn.Module] = None,
     output_cross_attentions: bool = False,
 ) -> Union[torch.FloatTensor, Tuple[torch.FloatTensor, torch.FloatTensor]]:
+    if speaker_embeddings is None:
+        logger.warning_once(
+            """`speaker_embeddings` is not specified, which will probably generate unintelligible speech.
+                    Please, specify a `speaker_embeddings`. For example, you can use a speaker embeddings by following
+                    the code snippet provided in this link:
+                    https://huggingface.co/datasets/Matthijs/cmu-arctic-xvectors
```

Comment:
Should we move this dataset to `microsoft` or `CMU` repo? Feels a bit weird having it under a personal namespace. 

Thoughts?

Reply:
We'll have to ask Matthijs 😅